class CustomerValidationException(Exception):
    pass


class CustomerNotFoundException(Exception):
    pass